crates/compiler/builtins/roc/Inspect.roc

```diff
@@ -92,3 +98,252 @@ inspect : val -> f where val implements Inspect, f implements InspectFormatter
 inspect = \val ->
     (@Inspector valFn) = toInspector val
     valFn (init {})
+
+# The current default formatter for inspect.
+# This just returns a simple string for debugging.
+# More powerful formatters will likely be wanted in the future.
+DbgFormatter := { data : Str }
```

Comment:
Not important right now, but I wonder if we should split this out into a separate module.

Reply:
Yeah, I debated the same, but ultimately thought it would be nicer to just have `Inspect.toDbgStr`. I am open to either, but that can be dealt with in follow up.